libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -466,13 +511,16 @@ MQTTStatus_t MQTT_GetDisconnectPacketSize( size_t * pPacketSize );
 /**
  * @brief Serialize an MQTT DISCONNECT packet into the given buffer.
  *
- * @param[out] pBuffer Buffer for packet serialization.
+ * The input #MQTTFixedBuffer_t.size must be at least the size returned by
```

Comment:
he input #MQTTFixedBuffer_t must be at least as large as he size returned by #MQTT_GetDisconnectPacketSize.